008_team_6_agent_design/008_team_6_agent_design.tex

```diff
@@ -1 +1,1021 @@
+
+To further investigate the importance of treaties, we override the initial decision of the collectivist agents to accept the treaty. This additional personal risk results in an overall benefit for the society, as seen in \Cref{fig:res_D}. Quantitatively, the number of deaths over time in this scenario is approximately 10 times smaller than in experiment D1 (\Cref{fig:res_D} (e)), accompanied by a uniformly positive utility over time (\Cref{fig:res_D} (f)). In addition, the initial acceptance of this new treaty also allows for the emergence of a stable distribution of social motives across the tower (\Cref{fig:res_D} (d)).
+
+Despite the presence of selfish (and even narcissistic) agents in the tower, they all follow the rule dictated by the treaties they signed whilst being collectivist. \footnote{Thus simulation enforces \textit{honest} agents that do not have the ability to break a treaty before the expiration date. We leave the concept of deception for future work.} An issue with this configuration is that the exact day when the agents are eating is not synchronized, leading to the selfish and narcissistic agents eating 2 food as soon as their health falls below the \textit{weakLevel}. We hence conclude that not waiting in the critical zone as collectivists do leads to a non-zero number of deaths.
+
+In addition, we can also see the effect of the reshuffle period on the social motives distribution in \Cref{fig:res_D} (d). The reshuffle period is 30 days in this case and we see that the reorganization of the tower leads to some of the narcissistic agents updating their social motive to revert to selfish.
+
+The main observation to draw from these results is in the terminal behaviour and the capability to reach a stable steady state, with the distribution of all agent types tending to a continuous value (approximately 40\% collectivist, 40\% selfish and 20\% narcissist). We propose that it is this enforcement of treaties that allows for otherwise self-interested agents to act selflessly, instead acting towards a common goal of collectivism. This re-stabilisation of the initially homogeneous collectivist society through treaties is so powerful that it can support a substantial proportion of narcissistic agents, who have in previous experiments (A4, C4) been sufficient for disrupting the stability of a system even in low concentrations. We thus conclude that the inclusion of localised governance allows agents to transcend the limitations of impulse due to pre-defined social behaviours and self organise to maintain a stable system that supports all behavioural types.
```

Comment:
```suggestion
The main observation to draw from these results is in the terminal behaviour and the capability to reach stability, with the distribution of all agent types tending to a continuous value (approximately 40\% collectivist, 40\% selfish and 20\% narcissist). We propose that it is this enforcement of treaties that allows for otherwise self-interested agents to act selflessly, instead acting towards a common goal of collectivism. This re-stabilisation of the initially homogeneous collectivist society through treaties is so powerful that it can support a substantial proportion of narcissistic agents, who have in previous experiments (A4, C4) been sufficient for disrupting the stability of a system even in low concentrations. We thus conclude that the inclusion of localised governance allows agents to transcend the limitations of impulsive behaviours due to pre-defined social behaviours and self-organise to maintain a stable system that supports all behavioural types.
```
the last sentence is too long in my opinion